### PR TITLE
[srpc] Update to 0.9.7

### PR DESCRIPTION
**Pull request by holmes1412**
Updates srpc, no changes to the portfile.

- #### What does your PR fix?
  Update srpc to the latest version. 

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  All. Yes.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes.

### CODE REVIEW

**Comment by FrankXie05** [comment]
@holmes1412 Please sign the [CLA](https://cla.opensource.microsoft.com/microsoft/vcpkg?pullRequest=26772).

File: ports/srpc/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "srpc",
-  "version-semver": "0.9.3",
+  "version-semver": "0.9.7",
```
Comment (FrankXie05) [inline ports/srpc/vcpkg.json:3]:
```suggestion
  "version-semver": "0.9.7",
  "license": "Apache-2.0",
```
Please add the license. :)

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Comment by FrankXie05** [comment]
Log from `config-x64-windows-dbg-err (2).log` https://dev.azure.com/vcpkg/public/_build/results?buildId=77968&view=artifacts&pathAsName=false&type=publishedArtifacts
````
CMake Error at CMakeLists.txt:57 (message):
  Workflow is neither installed not in third_party!
````
[config-x64-windows-dbg-err (2).log](https://github.com/microsoft/vcpkg/files/9581244/config-x64-windows-dbg-err.2.log)

**Comment by FrankXie05** [comment]
Pinging @holmes1412  for response. Is work still being done for this PR?

**Comment by FrankXie05** [comment]
This PR has been inactive for a long time, draft this PR.

**Comment by FrankXie05** [comment]
This PR has been inactive for a long time, please reopen it if there is any progress.

**Comment by holmes1412** [comment]
Thanks ! I will check the dependencies and make another PR after I fix the compile error. Thanks for your help again!

**Pull request closed without merging**